Tests/Unit/OpenIdConnectClientTest.php

```diff
@@ -182,7 +182,7 @@ public function getAccessTokenReturnsAccessTokenFromAuthorization(): void
         $clientId = 'the-client';
         $clientSecret = 'the-secret';
         $scope = 'some openid';
-        $authorizationId = Authorization::generateAuthorizationIdForClientCredentialsGrant($serviceName, $clientId, $clientSecret, $scope, []);
```

Comment:
This is needed in the v3 OAuth2 client… well, not strictly, as an empty array is the default value, but still.